README.md

```diff
@@ -5,10 +5,11 @@ virtual machine, and standard libraries that together enable Toit programs to ru
 
 ## Jaguar: Live reloading for the ESP32
 
-You can use Toit to develop, update, and restart your ESP32 applications in less than two seconds through
-[Jaguar](https://github.com/toitlang/jaguar). Jaguar is a small Toit program that runs on your ESP32.
-It uses the capabilities of the Toit virtual machine to let you update and restart your ESP32
-applications written in Toit over WiFi whenever your source files change. Once set up, it is as easy as:
+If you are developing for the ESP32, we recommend to use [Jaguar](https://github.com/toitlang/jaguar). It
+automatically downloads a Toit SDK. For host development, see below.
```

Comment:
It is easy to argue that 'a' is more correct than 'the' here, but it feels easier to understand if we just refer to it as the SDK even though there are multiple versions of it.

```suggestion
automatically downloads the Toit SDK. For host development, see below.
```